docs/manual/article-management/simple-tokens.de.md

```diff
@@ -0,0 +1,98 @@
+| --------------------| --------------------------------------------------------- | --------------------------- |
+| `##tstamp##`        | Zeitstempel                                               |                             |
+| `##flang##`         | Aktuell verwendete Sprache                                |                             |
+| `##domain##`        | Aktuelle Domain                                           | Newsletter                  |
+| `##link##`          | Link zum Newsletter                                       | Newsletter                  |
+| `##channels##`      | Abonnierter News-Channel                                   | Newsletter                  |
+
+
+## Wo können Simple Tokens eingesetzt werden?
+- Benutzer: Modultyp 'Registrierung'
+- Benutzer: Modultyp 'Passwort vergessen'
+- Newsletter: 'Newsletterleser', 'Abonnieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
+- Extensions z. B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads, MP_Forms
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
+
+## Weitere Einsatzzwecke:
+`datei_von_##tstamp##.pdf` für Dateien erzeugt datei_von_1650437899.pdf
+
+`files/data/##form_broschuere##.pdf` Pfad zur PDF-Datei mit individuellen Dateinamen
+
+Es ist ebenso möglich, Simple Tokens als Fallabfrage zu verwenden. Hierzu werden Simple Tokens nicht mehr mit `##` geschrieben, sondern mit öffnender und schließender Klammer `{}`
+
+Beispiel für das Notification-Center:
+
+```
+Anfrage für: ##form_herkunft##
+{if form_termine!=""}
+   Termin: ##form_termine##
+{endif}
+
+{if form_formtyp!="erweitert"}
+  Name: ##form_name##
+  E-Mail: ##form_email##
+  Telefon: ##form_telefon##
+  {else}
+  Vorname: ##form_vorname##
+  Name: ##form_name##
+  Straße: ##form_strasse##
+  PLZ: ##form_plz##
+  Ort: ##form_ort##
+{endif}
+```
+
+Beispiel für das Newsletter-Modul:
+
+```
+{if flang=="en"}
+  Your language is English.
+{elseif=="de"}
+  Deine Sprache ist Deutsch.
+{else}
+  Couldn't assign a language.
+{endif}
+```
+
+
+## Welche Operatoren können bei Bedingungen gesetzt werden?
+
+| Syntax      | Beschreibung                  |
+| ----------- | ------------------------------|
+| `==`        | Expliziter Vergleich          |
+| `!=`        | Ungleich                      |
+| `===`       | Strikt mit Typ-Konvertierung  |
+| `!==`       | Nicht identisch               |
+| `<`         | Kleiner als                   |
+| `>`         | Größer als                    |
+| `<=`        | Kleiner gleich                |
+| `>=`        | Größer gleich                 |
+
+Die Operatoren können mit oder ohne Leerzeichen gesetzt werden. Es funktioniert daher sowohl `{if form_name != ""}` als auch `{if form_name!=""}`.
+
+Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```

Comment:
That only works as of Contao 4.10 and Symfony Expression Language.

Reply:
`{if form_name != ""}` und `{if form_name!=""}` funktioniert auch in der 4.9

DIe Operatoren `||` und `&&` funktionieren tatsächlich noch nicht in der 4.9

Reply:
Yes, that's what I'm saying.

Reply:
done